reference/employer/openapi.yaml

```diff
@@ -2499,6 +2578,98 @@ components:
                   - family
                 description: Type for HSA contribution limit if the benefit is a HSA.
                 nullable: true
+    BenefitFeaturesAndOperations:
+      type: object
+      nullable: true
+      properties:
+        supported_features:
+          $ref: '#/components/schemas/BenefitFeature'
+        supported_operations:
+          $ref: '#/components/schemas/SupportPerBenefitType'
+    AuthenticationMethod:
+      type: object
+      properties:
+        type:
+          type: string
+          description: The type of authentication method.
+          enum:
+            - assisted
+            - credential
+            - api_token
+            - oauth
+        benefits_support:
```

Comment:
we're returning `supported_fields` in the response now so we should add that here as well, and set both `benefits_support` and `supported_fields` to be nullable

Reply:
Yup they should both be nullable and I think you already fixed that